prov/rxm/src/rxm_conn.c

```diff
@@ -370,7 +370,12 @@ struct util_cmap *rxm_conn_cmap_alloc(struct rxm_ep *rxm_ep)
 	int ret;
 
 	len = rxm_ep->msg_info->src_addrlen;
-	name = malloc(len);
+	name = calloc(1, len);
```

Comment:
Why the change? Other caller don;'t zero out the buffer.

Reply:
If the underlying provider fills not entire buffer, we should know the boundaries of the buffer that was filled.
Yes, I know we have a length. But I guess it would add some sort of safety

Reply:
It doesn't matter since rxm_conn_cmap_alloc is memdup'ing src_addrlen bytes from that buffer.

And that also means that name is leaked when rxm_conn_cmap_alloc exits.

Reply:
@fzago-cray Thanks for finding this. It's really memory leak. I'll fix this!

Reply:
I've updated to fix memory leak

Reply:
Looks good. Thanks.

Reply:
@fzago-cray Thank you too! it was a good catch!